tests/ft_rust.vader

```diff
@@ -1,5 +1,11 @@
 Include: include/setup.vader
 
+Before (create src directory):
+  call mkdir('src')
+
+After (cleanup src directory):
+  !rmdir src
+
```

Comment:
I've not looked at the tests closely, so maybe it actually fixes something there even..